src/webui/www/private/index.html

```diff
@@ -4,28 +4,28 @@
 <head>
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=10" />
-    <title>qBittorrent ${VERSION} QBT_TR(Web UI)QBT_TR[CONTEXT=OptionsDialog]</title>
+    <title>qBittorrent Web UI</title>
```

Comment:
Why did you remove translation here?

Reply:
The `<title>` value will be overwritten by js code in client.js, so I thought we can save a little bit of processing here, or should I revert?

Reply:
>The <title> value will be overwritten by js code in client.js, 

Ok.